### DISCUSSION TITLE
In desktop browsers, allow users to connect with metamask browser extension

### DISCUSSION THREAD

**Discussion by timqian** [WalletConnect/walletconnect-monorepo#1001] [Ideas]
In the desktop browsers, It will be cool if WallectConnect can pop up metamask chrome extension directly and allow users to connect.

Benefits
- For users: they don't need to take out their phone and scan a QR code
- For developers: most dapps today need to implement two kinds of logic: connect wallet with WC or connect wallet with MetaMask. If connect with metamask extension can be supported. Developer will only need to use WC alone

**Comment by jacobsalomon** [comment]
Can't WalletConnect work with browser extension wallets currently if the wallet extension has decided to support the protocol?

https://github.com/WalletConnect/walletconnect-monorepo/discussions/1370
